src/lib/helpers.js

```diff
@@ -10,18 +10,17 @@ const getCourseImageUrl = courseData => {
   }${courseNameParts[courseNameParts.length - 2].charAt(0)}${courseNameParts[
     courseNameParts.length - 1
   ].slice(2)}.jpg`
-  let courseImageUrl = "images/course_image.jpg"
-  courseData["course_files"].forEach(media => {
+  const courseImageMedia = courseData["course_files"].filter(media => {
```

Comment:
Actually, you can use `find` here if you want to get the first item which matches a condition